assets/src/js/admin/paypal-commerce/index.js

```diff
@@ -103,51 +117,158 @@ window.addEventListener( 'DOMContentLoaded', function() {
                     },
                 };
                 const paypalErrorQuickHelp = document.getElementById( 'give-paypal-onboarding-trouble-notice' );
+                const ajaxRequest = () => {
```

Comment:
Making this an `async` function would clean up the subsequent code quite a bit by allowing the use of `await`.

Reply:
fixed: https://github.com/impress-org/givewp/pull/6867/commits/f1deaa8a034da48b5e0482833c1ab7299f94d38a